mac/KeymanEngine4Mac/KeymanEngine4Mac/KME/OnScreenKeyboard/KeyView.m

```diff
@@ -49,9 +49,12 @@ - (id)initWithFrame:(NSRect)frame {
         if ([_caption respondsToSelector:@selector(setLineBreakMode:)]) {
             [_label setLineBreakMode:NSLineBreakByClipping];
         } // There might be some problem not calling this, but it seems to be okay as far as I can tell
+        
+        CGFloat fontSize = labelFrame.size.height*0.3;
```

Comment:
I wonder if we can make the `0.3` and `0.45` values constants at the top of this file, especially given we use `0.3` in two places?